### ISSUE TITLE
Install on IIS: getting a 404

### ISSUE THREAD

**Issue by Myrskyvaris** [snipe/snipe-it#4128]
Let me preface this by saying I have never installed & configured IIS or PHP before. 

I think I have IIS on Server 2016 installed & configured correctly, and I think I have Snipe It and MySql installed correctly, and my DNS seems to be correctly configured. 

I configured web folder permissions, edited .env, ran composer in the snip-it folder. 

I believe I have all the PHP extentions installed, but it's not clear or not if I have the PHPZIP extension.

When I go to http://localhost on the Snipe It server, and when I go to the URL  (assets.glenoaks.edu) I get a 404. 

The only issue I think I may have picked up in importing .htaccess. I get a 0 of 0 rule(s) was converted successfully. I'm assuming then that there are no rules in .htaccess and I should have added them?

- [Yes] I have enabled debug mode 
- [Yes] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

#### Provide answers to these questions:

- [Fresh] Is this a fresh install or an upgrade? 
- [4.04] Version of Snipe-IT you're running
- [5.3 & 5.6] Version of PHP you're running
- [mariadb-10.0.14-winx64] Version of MySQL/MariaDB you're running
- [2016/10] What OS and web server you're running Snipe-IT on
- [Manual] What method you used to install Snipe-IT (install.sh, manual installation, docker, etc)
- [Still a 404] WITH DEBUG TURNED ON, if you're getting an error in your browser, include that error
- [N/A] What specific Snipe-IT page you're on, and what specific element you're interacting with to trigger the error
- [N/A] If a stacktrace is provided in the error, include that too.
- [N/A] Any errors that appear in your browser's error console.
- [No] Confirm whether the error is reproduceable on the demo: https://snipeitapp.com/demo.
- [No Snip It logs generated]  Include any additional information you can find in `app/storage/logs` and your webserver's logs.
- [No error messages]  Include what you've done so far in the installation, and if you got any error messages along the way.
- [No] Indicate whether or not you've manually edited any data directly in the database

IIS Log
2017-10-03 18:13:30 ::1%0 54869 ::1%0 80 HTTP/1.1 GET / - 404 - NotFound -
2017-10-03 18:31:00 ::1%0 54885 ::1%0 80 HTTP/1.1 GET / - 404 - NotFound -
2017-10-03 18:31:04 ::1%0 54886 ::1%0 80 HTTP/1.1 GET / - 404 - NotFound -
2017-10-03 18:31:18 10.1.14.113 57540 172.16.8.17 80 HTTP/1.1 GET /snipe-it/index.php - 404 - NotFound -
2017-10-03 18:31:20 10.1.14.113 57542 172.16.8.17 80 HTTP/1.1 GET / - 404 - NotFound -
2017-10-03 18:35:09 ::1%0 55042 ::1%0 80 HTTP/1.1 GET / - 404 - NotFound -
2017-10-03 18:35:12 ::1%0 55043 ::1%0 80 HTTP/1.1 GET / - 404 - NotFound -
2017-10-03 18:35:17 10.1.14.113 57881 172.16.8.17 80 HTTP/1.1 GET / - 404 - NotFound -
2017-10-03 18:35:18 10.1.14.113 57883 172.16.8.17 80 HTTP/1.1 GET / - 404 - NotFound -
2017-10-03 18:35:24 10.1.14.113 57901 172.16.8.17 80 HTTP/1.1 GET / - 404 - NotFound -
2017-10-03 18:35:30 10.1.14.113 57903 172.16.8.17 80 HTTP/1.1 GET /snipe-it/ - 404 - NotFound -
2017-10-03 18:35:49 10.1.14.113 57953 172.16.8.17 80 HTTP/1.1 GET / - 404 - NotFound -
2017-10-03 18:35:49 10.1.14.113 57955 172.16.8.17 80 HTTP/1.1 GET /favicon.ico - 404 - NotFound -
2017-10-03 18:38:51 ::1%0 55048 ::1%0 80 HTTP/1.1 GET / - 404 - NotFound -
2017-10-03 18:41:48 ::1%0 55054 ::1%0 80 HTTP/1.1 GET / - 404 - NotFound -
2017-10-03 18:42:08 10.1.14.113 58389 172.16.8.17 80 HTTP/1.1 GET / - 404 - NotFound -
2017-10-03 18:43:30 ::1%0 55056 ::1%0 80 HTTP/1.1 GET / - 404 - NotFound -
2017-10-03 18:45:22 ::1%0 55060 ::1%0 80 HTTP/1.1 GET /install.php - 404 - NotFound -
2017-10-03 18:45:38 ::1%0 55062 ::1%0 80 HTTP/1.1 GET /snipe-it - 404 - NotFound -
2017-10-03 18:59:57 ::1%0 55068 ::1%0 80 HTTP/1.1 GET / - 404 - NotFound -
2017-10-03 19:20:02 10.1.14.113 60562 172.16.8.17 80 HTTP/1.1 GET / - 404 - NotFound -
2017-10-03 19:20:04 10.1.14.113 60564 172.16.8.17 80 HTTP/1.1 GET / - 404 - NotFound -
2017-10-03 19:35:34 ::1%0 55107 ::1%0 80 HTTP/1.1 GET / - 404 - NotFound -
2017-10-03 19:35:42 10.1.14.113 61687 172.16.8.17 80 HTTP/1.1 GET / - 404 - NotFound -
2017-10-03 19:35:42 10.1.14.113 61691 172.16.8.17 80 HTTP/1.1 GET / - 404 - NotFound -

**Comment by snipe** [comment]
You shouldn't need the htaccess, since you have a web.config in your public directory.

**Comment by Myrskyvaris** [comment]
Thanks - in that case I will look for the problem elsewhere.

**Comment by madd15** [comment]
@Myrskyvaris Did you import the .htaccess file from the public directory or the base dir?

**Comment by snipe** [comment]
@madd15 holy cow! Long time no see! <3

If they are using IIS, do they still need to import the htaccess stuff? I thought that's what the web.config was for.

**Comment by madd15** [comment]
@snipe yeah been awhile.

Yes the web.config should provide the URL rewrite rules to IIS, I have tested this again just now with a fresh web.config and it does appear to work as intended (no .htaccess import required).

@Myrskyvaris is your physical path correct? eg `C:\inetpub\wwroot\snipe-it\public`

**Comment by madd15** [comment]
also `As of PHP 5.3 this extension is built-in`

**Comment by Myrskyvaris** [comment]
@madd15 my physical oath is: C:\inetpub\wwwroot\snipe-it\public

**Comment by Myrskyvaris** [comment]
It seems the consensus is that I don't need to worry about htaccess and PHPZIP is now built in.

**Comment by Myrskyvaris** [comment]
Okay, so what I found was that I set the site binding to assets.glenoaks.local instead of assets.glenoaks.edu. I have no excuse for this sad behaviour!

The setup page then complained that Debug was on, so I switched it off. 

During the DB table creation process, I got a "whoops something went wrong" page, but after refreshing the message was that the tables exist and that nothing can be migrated. 

The rest of the process seemed to flow smoothly and I'm testing the application now!
